src/kOS/Function/Persistence.cs

```diff
@@ -6,9 +6,136 @@
 using kOS.Safe.Utilities;
 using kOS.Serialization;
 using System;
+using KSP.IO;
+using kOS.Safe;
+using kOS.Safe.Compilation;
+using System.Collections.Generic;
 
 namespace kOS.Function
 {
+    /*
+     * A couple of syntaxes from kRISC.tpg were deprecated when subdirectories where introduced. It will be possible to
+     * remove these function below as well any metions of delete/rename file/rename volume/copy from kRISC.tpg in the future.
+     */
+    [Function("copy_deprecated")]
+    public class FunctionCopyDeprecated : FunctionBase
+    {
+        public override void Execute(SharedObjects shared)
+        {
+            PopValueAssert(shared, true);
+            PopValueAssert(shared, true);
+            PopValueAssert(shared, true);
+            AssertArgBottomAndConsume(shared);
+
+            throw new KOSDeprecationException("1.0.0", "`COPY FILENAME FROM VOLUMEID.` syntax", "`COPYPATH(FROMPATH, TOPATH)`", string.Empty);
```

Comment:
I'm not against the idea of breaking this entirely eventually, but maybe that should wait for a big major version number update, so we can break it at the same time as we break everything else.